trillian_map_api.proto

```diff
@@ -81,6 +82,14 @@ service TrillianMap {
   // For indexes that do not exist, the inclusion proof will use nil for the empty leaf value.
   rpc GetLeaves(GetMapLeavesRequest) returns(GetMapLeavesResponse) {}
   rpc SetLeaves(SetMapLeavesRequest) returns(SetMapLeavesResponse) {}
-  rpc GetSignedMapRoot(GetSignedMapRootRequest) returns(GetSignedMapRootResponse) {}
-  rpc GetSignedMapRootByRevision(GetSignedMapRootByRevisionRequest) returns(GetSignedMapRootResponse) {}
+  rpc GetSignedMapRoot(GetSignedMapRootRequest) returns(GetSignedMapRootResponse) {
```

Comment:
Similarly as the comment above, I'd go for:

GetSignedMapRoot:
GET **/v1beta1/maps/{map_id}/roots:latest**

GetSignedMapRootByRevision:
GET **/v1beta1/maps/{map_id}/roots:revision**

(I'm working under the assumption that {revision} is not the logical ID for map roots.)